### DISCUSSION TITLE
v23.0.0-rc.2

### DISCUSSION THREAD

**Discussion by thaJeztah** [moby/moby#44807] [Announcements]
This is a pre-release of the upcoming 23.0.0 release. 

Pre-releases are intended for testing new releases: **only install in a test environment!**

```bash
curl -fsSL https://get.docker.com -o get-docker.sh
sudo CHANNEL=test sh get-docker.sh
```

> Starting with the 23.0.0 release, we're moving away from using CalVer versioning,
> and use versioning that's using SemVer _format_. Changing the version format is
> a stepping-stone towards go module compatibility, but the repository does not
> yet use go-modules, and still requires using a "+incompatible" version; we'll
> be working towards go module compatibility in a future release.

### Known issues:

- There is no changelog yet; an overview of pull requests included in this release can be found on GitHub:
    - docker cli: [all pull requests for 23.0.0](https://github.com/docker/cli/pulls?q=is%3Apr+milestone%3A23.0.0+is%3Amerged) / [all "changelog" pull requests for 23.0.0](https://github.com/docker/cli/pulls?q=is%3Apr+milestone%3A23.0.0+label%3Aimpact%2Fchangelog+is%3Amerged)
    - docker engine: [all pull requests for 23.0.0](https://github.com/moby/moby/pulls?q=is%3Apr+milestone%3A23.0.0+is%3Amerged) / [all "changelog" pull requests for 23.0.0](https://github.com/moby/moby/pulls?q=is%3Apr+milestone%3A23.0.0+label%3Aimpact%2Fchangelog+is%3Amerged)
- Static binaries (`.tar.gz`) are no longer include cli plugins; use the deb/rpm packages, or manual installation instructions from the [buildx github repository](https://github.com/docker/buildx#manual-download) and  [docker compose github repository](https://github.com/docker/compose/#where-to-get-docker-compose)
- Static binaries for `dockerd` are broken on `armhf` and `armel` (32-bit).
- There are no packages available yet for the s390x and ppc64le architectures
- The `docker-buildx-plugin` `.deb` package for  `0.10.0-rc2` had an incorrectly formatted version-string, causing it to be installed instead of the `0.10.0` (non-pre-release) version. To install v0.10.0 of buildx instead of the 0.10.0-rc2 pre-release, specify the version to install. For example, if you're on Ubuntu 22.04 "jammy", use `apt-cache` to find the available versions of the `docker-buildx-plugin` package:

    ```bash
    apt-cache madison docker-buildx-plugin
    docker-buildx-plugin | 0.10.0-rc2~ubuntu-jammy | https://download-stage.docker.com/linux/ubuntu jammy/test amd64 Packages
    docker-buildx-plugin | 0.10.0-1~ubuntu.22.04~jammy | https://download-stage.docker.com/linux/ubuntu jammy/test amd64 Packages
    docker-buildx-plugin | 0.9.1~ubuntu-jammy | https://download-stage.docker.com/linux/ubuntu jammy/test amd64 Packages
    ...
    ```
    
    In this case, the version to specify is `0.10.0-1~ubuntu.22.04~jammy`;
    
    ```bash
    apt-get install docker-buildx-plugin=0.10.0-rc2~ubuntu-jammy
    Reading package lists... Done
    Building dependency tree... Done
    Reading state information... Done
    The following packages will be DOWNGRADED:
      docker-buildx-plugin
    0 upgraded, 0 newly installed, 1 downgraded, 0 to remove and 82 not upgraded.
    Need to get 25.9 MB of archives.
    After this operation, 34.8 kB of additional disk space will be used.
    Do you want to continue? [Y/n]
    ```
- There may be bugs! :warning:

Bugs and regressions can be reported in these issue trackers:

- Related to the CLI: https://github.com/docker/cli/issues
- Related to the Docker Engine https://github.com/moby/moby/issues

When reporting issues, include `[23.0.0-rc]` in the issue title

<hr /><em>This discussion was created from the release <a href='https://github.com/moby/moby/releases/tag/v23.0.0-rc.2'>v23.0.0-rc.2</a>.</em>

**Comment by Romain-Geissler-1A** [comment]
Hi,

Do you know roughly when the final first release 23 will be out ? Is it planned for January or more February/March ?

Thanks,
Romain

**Reply by thaJeztah** [reply]
We don't have a fixed date yet (It's done when it's done), but should be soon now; we're working on some final bits (mostly around packaging), after which we should be ready to release.
